infra/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+# Inspired by https://docs.docker.com/compose/django/
```

Comment:
Ik twijfelde even of er wel een ``docker-compose.yml`` moet zijn voor de 3 componenten samen. Voor developers die het willen uitproberen lijkt me dit echter uitermate fijn.

Alleen even oppassen dat we documentatie-wise niet deze 3 als 1 ding gaan zien, want dan gaat het effect van opsplitsen verrloren.

Reply:
Klopt, het idee van deze documentatie is dat het absolute minimum is. Ik zal verwijzen naar de uitgebreide documentatie per component.